src/App.css

```diff
@@ -16,24 +16,28 @@ a {
   text-decoration: none;
   display: flex;
 }
-[theme="light"] {
+[theme="dark"] {
   --primary-color: #ff8c44;
   --primary-bg-color: #151515;
   --secondary-bg-color: rgb(34, 31, 31);
   --primary-theme-color: white;
   --secondary-theme-color: black;
+  --white-color: white;
+  --black-color: black;
   --font-size-XX-large: clamp(3rem, 12vw, 10rem);
```

Comment:
I like that you are using clamp function

Reply:
ThankYou